### DISCUSSION TITLE
hakoniwa-ros2simにおけるUnityバージョンのトラブル報告

### DISCUSSION THREAD

**Discussion by YusukeKato** [toppers/hakoniwa#167] [Q&A] [answered]
hakoniwa-ros2simにおけるトラブル報告です。

READMEにはUnity Editor 2021.3.0f1をインストールするように書かれていますが、下記の画像のように2021.3.7f1をインストールするように警告が出ました。
https://github.com/toppers/hakoniwa-ros2sim#install-unity

![Unityプロジェクトを開くときの警告](https://github.com/toppers/hakoniwa/assets/12107761/ab7175d3-b82b-46ef-b3ee-071becf95fa2)

Unity Editor 2021.3.7f1をインストールすると警告なしでUnityプロジェクトを開くことができました。

また、こちらのissueで似たような話がされていますが、別件だと思われます。
https://github.com/toppers/hakoniwa-ros2sim/issues/4

**Comment by tmori** [comment, marked as answer]
> hakoniwa-ros2simにおけるトラブル報告です。
> 
> READMEにはUnity Editor 2021.3.0f1をインストールするように書かれていますが、下記の画像のように2021.3.7f1をインストールするように警告が出ました。 https://github.com/toppers/hakoniwa-ros2sim#install-unity
> 
> ![Unityプロジェクトを開くときの警告](https://user-images.githubusercontent.com/12107761/271769841-ab7175d3-b82b-46ef-b3ee-071becf95fa2.png)
> 
> Unity Editor 2021.3.7f1をインストールすると警告なしでUnityプロジェクトを開くことができました。
> 
> また、こちらのissueで似たような話がされていますが、別件だと思われます。 [toppers/hakoniwa-ros2sim#4](https://github.com/toppers/hakoniwa-ros2sim/issues/4)

ご連絡ありがとうございます！
Unityは日々バージョンが上がっておりまして、ドキュメントが追従できない状態でした・・。
基本的に、上位互換と思いますので、最新のUnityをご利用いただければ動作すると思います。

**Reply by YusukeKato** [reply]
回答ありがとうございます！
承知しました。
箱庭、楽しませていただいております。
今後もよろしくお願いいたします。

**Comment by YusukeKato** [comment]
すみません、オープンのままになっていたのでクローズさせていただきます。
ご対応ありがとうございました。
